app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (NotWearingMaskError,
+                        NotVaccinatedError,
+                        OutdatedVaccineError)
+
+
+def go_to_cafe(friends, cafe) -> str:
+    total = 0
+
+    for i in range(len(friends)):
+        try:
+            cafe.visit_cafe(friends[i])
+        except (NotVaccinatedError, OutdatedVaccineError):
```

Comment:
To be more concise, this should be written as:
```python
except VaccineError:
```
In this way every subclass of `VaccineError` will be caught, that was the purpose of creating `NotVaccinatedError` and `OutdatedVaccineError` as subclasses of `VaccineError`